src/integrationtests/java/util/IntegrationTestBase.java

```diff
@@ -194,42 +200,45 @@ protected JsonNode getJSONFromFile(String filePath) throws IOException {
     /**
      * Wait for certain amount of events to be processed.
      *
-     * @param eventsCount - An int which indicated how many events that should be
-     *                    processed.
+     * @param eventsCount - An int which indicated how many events that should be processed.
      * @return
      * @throws InterruptedException
      */
     protected void waitForEventsToBeProcessed(int eventsCount) throws InterruptedException {
         // wait for all events to be processed
-        long stopTime = System.currentTimeMillis() + 60000;
+        long stopTime = System.currentTimeMillis() + 30000;
```

Comment:
Could you add a constant e.g. `SECONDS_30`? It would make it easier to distinguish between 3000 and 30000.

Reply:
Done.